tests/unittests/Foundation/NSCompoundPredicateTests.m

```diff
@@ -388,7 +388,7 @@
 }
 
 TEST(Foundation, NSCompoundPredicate_copy) {
-    NSPredicate* predicate = [[NSPredicate alloc] init];
+    NSPredicate* predicate = [NSPredicate predicateWithFormat:@"10 >= 30"];
```

Comment:
What about the other tests in this file?  Looks like there are other failures as well, could you please fix all the related issues?